src/status_im/android/core.cljs

```diff
@@ -44,13 +44,16 @@
                        "keyboardDidShow"
                        (fn [e]
                          (let [h (.. e -endCoordinates -height)]
+                           (dispatch [:set :tab-bar-visible? false])
                            (when-not (= h @keyboard-height)
                              (dispatch [:set :keyboard-height h])
                              (dispatch [:set :keyboard-max-height h])))))
          (.addListener react/keyboard
                        "keyboardDidHide"
-                       #(when-not (= 0 @keyboard-height)
-                          (dispatch [:set :keyboard-height 0])))
+                       (fn [e]
```

Comment:
Seems like we are not using the `e` parameter, could you please change this to `_` ?